pkg/logql/metrics_test.go

```diff
@@ -111,7 +111,7 @@ func TestLogLabelsQuery(t *testing.T) {
 	})
 	require.Equal(t,
 		fmt.Sprintf(
-			"level=info org_id=foo traceID=%s latency=slow query_type=labels length=1h0m0s duration=25.25s status=200 label=foo throughput=100kB total_bytes=100kB total_entries=12\n",
+			"level=info org_id=foo traceID=%s latency=slow query_type=labels length=1h0m0s duration=25.25s status=200 label=foo splits=0 throughput=100kB total_bytes=100kB total_entries=12\n",
```

Comment:
I see we have test only for LabelQuery and SeriesQuery here. Good to add a test for `InstantQuery` as well? to see both `splits` and `shard` appear?

Reply:
I'm not entirely sure what these tests are actually testing? I just updated it to it wasn't failing but it seems strange to me to have a test that's validating the output of a log line?

I can see some arguments for this but we don't really have any contracts on log lines so I'm not sure we really need these tests?

Reply:
I'd say that we should probably never test log lines